Automattic/woocommerce-payments
# Remove standard and high fraud protection settings.

Fixes https://github.com/Automattic/woocommerce-payments-server/issues/3212


#### Changes proposed in this Pull Request

Remove the options "standard" and "high". 

We've decided to remove the settings as we need to understand what our merchants need from these options. We'll revisit this after seeing how merchants configure the advanced setting.

paJDYF-6pl-p2#comment-17251 

<img width="1103" alt="Screenshot 2023-04-03 at 06 47 47" src="https://user-images.githubusercontent.com/1713474/229416932-ee0bd6c3-f637-451f-acf0-686996b6bdeb.png">

#### Testing instructions
- Visit the fraud protection settings and make sure that you no longer see the standard and high options. You should only see basic and advanced.
- Chek to see if I'm missing anything that could break the feature. I removed the obvious pieces but I'm not 100% that I touched everything as I wasn't involved with building and reviewing the feature.
- Reset the welcome tour and ensure the new protection level step no longer shows "standard". It should now show  "from basic to advance". 
<img width="396" alt="Screenshot 2023-04-03 at 07 17 54" src="https://user-images.githubusercontent.com/1713474/229418057-2e381e05-0df6-4eaf-ad48-915f79d56558.png">


-------------------


**Post merge**



- [] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.
